### DISCUSSION TITLE
可以出一期关于单元测试的视频吗?

### DISCUSSION THREAD

**Discussion by dsjzazs** [BYJRK/DotNet-Discussions#44] [Q&A]
我是做工控软件的,一直想要在项目上用这个东西,就是不知道要怎么写. 比如软件有很多需要连接串口,TCP的这种,单元测试要怎么做

**Comment by BYJRK** [comment]
这个感觉不太好找例子，也不太容易深入介绍。你可以自己去看一看 xUnit、NUnit、MSTest 这些的教程，选择一个自己觉得合适的测试工具
